tests/addresslib/parser_mailbox_test.py

```diff
@@ -153,7 +156,6 @@ def test_display_name():
                           EmailAddress(' Bill Gates ', 'bill@microsoft.com'))
 
     # fail display-name quoted-string lax
-    run_mailbox_test('"Bill Gates"bill@microsoft.com', None)
```

Comment:
Valid address. Quoted string, atoms, and dot-atoms are always surrounded by optional whitespace according to the RFC so this is equivalent to `"Bill Gates" bill@microsoft.com`. This syntax is not in the RFC actually but the previous parser allowed display name with the address spec without `<>`s and we continued to allow that for backwards compatibility.